cloth-diapers/wash-routine.md

```diff
@@ -0,0 +1,139 @@
+to properly clean those diapers
+  + If you don't have enough diapers in your load, bulk it out with clothing or
+  small (hand) towels - *never use linens or large towels* because your diapers
+  can get wrapped up in them and won't wash properly
++ Wash your diapers at the temperature indicated by manufacturers, this is
+usually 40 degrees Celsius
+  + Once a month we wash at 60 degrees C for maintenance
+
+#### Dosing your detergent appropriately
+1. Powder detergent rinses off your diapers more easily than liquid detergent
+so ideally, **use powder**.
+2. Use 'real' detergent, not baby detergent or for sensitive skin as these do not
+wash as efficiently.
+3. Your detergent **must not have more than 5% soap content**
+4. Use the dose of detergent recommended for the total kilos of your machine,
+not the estimated amount of kilos in your diaper load (which remember, should
+  be 3 quarters of your machine anyway)
+5. Use the dose of detergent recommended for **heavily soiled** clothing (which
+  is an additional amount over the amount recommended for your machine size)
+6. Add the powder directly to the drum to get the most out of it and
+prevent gumming up your machine's tubes.
+
+Yes, this seems like a lot. But if you have the right programs and rinses in
+place, it's the best way to get those diapers clean.
+
+### 3. Final hot rinse, no detergent
+
+This should be 30 - 60 minutes long and be at the **same temperature as your
+main wash**.
+
+***This is just a rinse cycle, you don't want to add any more detergent here***.
+
+Again I would recommend doing this as a **separate cycle** so that you get a
+good long rinse rather than a short one tacked onto the end of your main
+program. We use the same program for the final rinse as we do for the prewash,
+we just make sure we change the temperature.
+
+**Your diapers shouldn't smell of anything** (not even laundry detergent,
+  except sometimes a little faintly) when you get to the end of this routine!
+
+
+## OUR Wash Routine
+
+We tend to wash at the end of every 3rd day, or at the end of every 2nd in the
+hot Summer to prevent smells.
+If we're washing more often (end of every second day) I always make sure I have
+small towels on hand to bulk out the load so it's 3 quarters full.
+
+These are the programs and steps for our LG washing machine, in a _soft_ water area.
+
+### 1. Cold rinse, no detergent
+
+Machine steps:
++ Algodão (cotton)
++ Turbowash (quickwash version of the program selected above)
++ Bring temperature down to Zero
+
+### 2. Hot main wash, full detergent, extra rinses
+
+// Detergent quantities go here.
+
+Machine steps:
++ Algodão (cotton)
++ Soil level: heavy (max)
++ 40 degrees (60 once a month as needed)
++ Extra rinse x 2
+
+
+### 1. Hot rinse, no detergent
+
+Machine steps:
++ Algodão (cotton)
++ Turbowash (quickwash version of the program selected above)
+
+
+<!-- They depend on:
++ The hardness of your water
```

Comment:
this commented-out section is worth including. guessing it's work-in-progress. 💭

Reply:
I just thought it was a bit too overwhelming for people, i.e. too many things can 'go wrong'.